dvc/serialize.py

```diff
@@ -35,10 +36,10 @@ def _get_out(out):
     res = OrderedDict()
     if not out.use_cache:
         res[PARAM_CACHE] = False
-    if out.metric:
-        res[PARAM_METRIC] = True
-    elif out.persist:
+    if out.persist:
         res[PARAM_PERSIST] = True
+    if out.plot and isinstance(out.plot, dict):
+        res.update(out.plot)
```

Comment:
Can you please point me where this is being set as dict? I couldn't find it. :confused:

Reply:
@skshetry It is actually not set anywhere programmatically (we might introduce `dvc plots modify` later, but for now it should be set by-hand).